charts/telegraf/templates/deployment.yaml

```diff
@@ -21,6 +21,10 @@ spec:
 {{ toYaml .Values.podAnnotations | indent 8 }}
         {{- end }}
     spec:
+{{- if .Values.securityContext }}
+      securityContext:
+{{ toYaml .Values.securityContext | indent 8 }}
+{{- end }}
```

Comment:
Please see and replicate https://github.com/helm/helm/blob/273d0364be30e3bbaa73041dceea2a338f666abe/pkg/chartutil/create.go#L269-L273 we should have both `podSecurityContext` and `securityContext` values

Reply:
to be honest i actually took from https://github.com/influxdata/helm-charts/blob/master/charts/influxdb/templates/statefulset.yaml if you can show me there where it is there? I cannot see it, I was mearly aligning with capability avaialble in the influxdb chart,

Reply:
touché

Reply:
mind you we wont be able to add without breaking because securityContext would go from meaning pod security context to container security context

Reply:
anyways not a blocker